artshow/register.py

```diff
@@ -38,6 +38,8 @@ def main(request):
         artist_form = ArtistRegisterForm()
         agreement_form = AgreementForm()
 
+    artists = Artist.objects.viewable_by(request.user)
     return render(request, "artshow/manage_register_main.html",
-                  {"artist_form": artist_form,
+                  {"artists": artists,
```

Comment:
Is the idea that it shows them their current registrations?

Should we have a more direct modal to add a registration if you already have one?

Reply:
This change adds the list of existing registrations at the top of the registration form:
 
![Screenshot](https://github.com/anthroarts/artshow-jockey/assets/25424176/e603c23f-a31a-4e70-bd1f-49fe30801907)

This is the same list that shows up if they just log into the management page instead of clicking on the "Register" link:

![Screenshot](https://github.com/anthroarts/artshow-jockey/assets/25424176/924ed9e7-78b4-4b18-9b38-947f0ea82d12)

That page also provides a link to the registration form so if they do want to add another profile they can. Hopefully this is helps clear up the confusion that led to duplicate registrations last year.